MauiProgram.cs

```diff
@@ -1,20 +1,28 @@
-﻿using Microsoft.Extensions.Logging;
+﻿using DotNetMaui.Services;
+using Microsoft.Extensions.Logging;
 
 namespace DotNetMaui;
 
 public static class MauiProgram
 {
+	public static bool DidCrashOnLastRun { get; private set; }

```

Comment:
What's the purpose of this prop?
We have an option in some SDKs that we use to get the value from a native layer. But I dunno a usecase to have it in the app as a rop

Reply:
This was going to be set if OnCrashedLastRun happened and then open the feedback window when the first screen was available (the event from Sentry can fire before the UI is ready).  It just isn't implemented yet.  I was going to go all out of features.